relay-node/service/src/chain_spec.rs

```diff
@@ -294,7 +310,14 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
         vec![
             // TBD
```

Comment:
I think here we can keep same addresses and allocations as in the current testnet